### PR TITLE
#3 review command line help

### PR DESCRIPTION
**Pull request by TimKam**

### CODE REVIEW

File: bin/binge.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ program
 program
     .command('graph')
     .description(
-        'Prints the local-package tree, and layer topology information'
+        'Prints the local-package tree, and layer topology information.'
```
Comment (rstiles) [inline bin/binge.js]: `local-package` -> `local package`

File: bin/binge.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ program
 program
     .command('nuke [target]')
     .description(
-        'Removes target folder or file from each package (default node_modules)'
+        'Removes the target folder or file from each packag. Defaults to node_modules.'
```
Comment (rstiles) [inline bin/binge.js]: Typo: `packag`

File: bin/binge.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ program
 
 program
     .command('cache-clean')
-    .description('cleans the build and install cache')
+    .description('Cleans the build and installs the cache.')
```
Comment (cristovaoth) [inline bin/binge.js]: Here what I wanted to say was really: clean the build cache and clean the install cache. So it doesn't install, it cleans the install cache :)

File: bin/binge.js [hunk 4]
```diff
@@ -120,7 +120,7 @@ program
 program
     .command('touch [name] [version]')
     .description(
-        'Update one dependency to a specific version, and write the yarn.lock. Propagates changes to packages that share the same dependency (default simply write yarn.lock)'
+        'Updates a dependency to a specific version, and writes to yarn.lock. Propagates changes to packages that share the same dependency. Defaults to simply writing to yarn.lock)'
     )
```
Comment (cristovaoth) [inline bin/binge.js]: Remove the trailling parenthesis

Comment (cristovaoth) [inline bin/binge.js]: Remove the trailling parenthesi

**Review by cristovaoth — CHANGES_REQUESTED** [review]

**Pull request merged**